lib/database/models/user.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:objectbox/objectbox.dart';
+
+import 'coupon.dart' show Coupon;
+import 'resto.dart' show Resto;
+
+@Entity()
+class User {
```

Comment:
What do you think about making entities accessible by adding an entity constructor? https://docs.objectbox.io/entity-annotations#make-entity-data-accessible

Reply:
https://docs.objectbox.io/relations#improve-performance

Reply:
> What do you think about making entities accessible by adding an entity constructor? https://docs.objectbox.io/entity-annotations#make-entity-data-accessible

Sounds good, I don't know if adding the constructor affects the functionality of the database or not, which values does it suppose to put there? Do I have to add all the values there?

mmm a think of a value you don't add you it gonna take the default value you declare here. 
The only exception is the `id` that object box increases it always 

> https://docs.objectbox.io/relations#improve-performance
Say only applicable to `Java`/`Kotlin`